manifests/kustomize/env/cert-manager/platform-agnostic-standalone-tls/patches/cluster-role-scheduledworkflow.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why is this patch required? Is it not already present from manifests/kustomize/base/pipeline/ml-pipeline-scheduledworkflow-role.yaml?